src/index.jsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import * as serviceWorkerRegistration from './serviceWorkerRegistration';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import axios from "axios";
+
+import App from "./App";
+import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
+import reportWebVitals from "./reportWebVitals";
+
+axios.defaults.baseURL = process.env.REACT_APP_API_URL;
```

Comment:
I think it is better to create axios instance instead of altering the global axios module. I have implemented in my PR, which you can see [here](https://github.com/CS3216-collectify/collectify-frontend/blob/marcus2k/components/src/utils/server.js).